ansible/roles/fyrevm_provision/tasks/fyrevm_provision.yml

```diff
@@ -56,25 +58,25 @@
       minutes: 2
 
   - name: get Fyre request status
+    no_log: "{{ noLog }}"
     uri:
-      url: "{{ fyre_requeststatusurl }}{{ (buildstatus.json).request_id }}"
+      url: "{{fyre_clusterstatusurl}}"
       method: "POST"
```

Comment:
is this still a POST?

Reply:
Fyre api doc's are ambiguous.  Seems it can be GET or POST.  I switched to GET per the example, versus the generic doc.

Reply:
fixed -> GET